app/views/play/CampaignView.coffee

```diff
@@ -668,7 +669,7 @@ module.exports = class CampaignView extends RootView
     console.error "CampaignView hero update couldn't find hero slug for original:", hero
 
   onClickPortalCampaign: (e) ->
-    campaign = $(e.target).closest('.campaign')
+    campaign = $(e.target).closest('.campaign, .beta-campaign') 
```

Comment:
Learned something from @phoenixeliot: you can use `e.currentTarget` to get the element that the event was attached to, rather than having to use `.closest()` in case a child element was clicked.